assets/schema_input.json

```diff
@@ -45,6 +33,12 @@
                 "exists": true,
                 "errorMessage": "Run directory must be a path",
                 "meta": ["rundir"]
+            },
+            "tags": {
+                "type": "string",
+                "pattern": "^\\\"([A-Za-z0-9_-]+,\\ ?)*([A-Za-z0-9_-]+)\\\"$",
+                "errorMessage": "Tags must consist of numbers, letters, underscores or dashes and must be provided as a comma-separated list flanked by a pair of double-quotes, e.g. \"patient_01, lane1, pos-CTRL_2\".",
```

Comment:
Why did you opt for exactly this specification? 

We basically need the quotes, because we ask for the same delimiter for the tags and the sample sheet. This may still cause issues when parsing. (_e.g. Excel in some versions auto-quotes all text fields_) and will be quite confusing for the users as well?

Is there any reason not to use e.g. a semi-colon or a colon as separator? Something like `project123:batch_B:location3` might work as well, no?

Reply:
If using e.g. a colon as a tag separator would help us get rid of the quotes, I'd be in favor!

Reply:
I'll change to columns :+1: